### ISSUE TITLE
All clients viewing a channel are scrolled to the bottom when a message is received in that channel

### ISSUE THREAD

**Issue by josephreed2600** [josephreed2600/bifrost#4]
To reproduce:

1. Log into Bifrost in two browsers
2. Open the same text channel in both browsers
3. Ensure that there are enough messages in the channel for one client (Client A) to scroll
4. In Client A, scroll the message area up so that the most recent message is not visible
5. In Client B, send a message

The expected behavior is that the message will be received by Client A and added to the list of messages displayed silently, without changing anything on the screen at the moment. When Client A is scrolled by the user down to the bottom, the message will be visible. The observed behavior is that Client A is immediately and automatically scrolled to the bottom of the message area, so the new message is shown immediately.

In simple terms, anyone sending a message in a channel will scroll everyone watching that channel to the bottom, but that shouldn't happen.
